packages/orderbook/test/js_api/gui.test.ts

```
@@ -1081,19 +1094,17 @@ ${dotrainWithoutVaultIds}
 			let result = await gui.chooseDeployment(testDotrain, 'other-deployment');
 			extractWasmEncodedData(result);
 
-			result = gui.getCurrentDeployment();
-			let deployment = extractWasmEncodedData<GuiDeploymentCfg>(result);
-			assert.equal(deployment.deployment.order.inputs[0].vaultId, undefined);
-			assert.equal(deployment.deployment.order.outputs[0].vaultId, undefined);
+			let deployment1 = extractWasmEncodedData<GuiDeploymentCfg>(gui.getCurrentDeployment());
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'deployment1' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)